API.adoc

```diff
@@ -0,0 +1,611 @@
+=== SMART COSMOS Edge User (DevKit) API Documentation
+
+== Authorities
+
+=== Tenant Authorities
+
+- https://authorities.smartcosmos.net/tenants/read
+- https://authorities.smartcosmos.net/tenants/update
+
+=== User Authorities
+
+- https://authorities.smartcosmos.net/users/create
+- https://authorities.smartcosmos.net/users/read
+- https://authorities.smartcosmos.net/users/update
+- https://authorities.smartcosmos.net/users/delete
+
+=== Role Authorities
+
+- https://authorities.smartcosmos.net/roles/create
+- https://authorities.smartcosmos.net/roles/read
+- https://authorities.smartcosmos.net/roles/update
+- https://authorities.smartcosmos.net/roles/delete
+
+== REST API
+
+<<tenants, *Available methods for tenants*>>
+
+|====================
+|Endpoint|REST API|Required Authority
+|<<tenantcreate1, Create>>|`POST /tenants`|none
+|<<tenantupdate1, Update>>|`PUT /tenants/{urn}`|https://authorities.smartcosmos.net/tenants/update
+|<<tenantread1, Find by URN>>|`GET /tenants/{urn}`|https://authorities.smartcosmos.net/tenants/read
+|<<tenantread2, Find by Name>>|`GET /tenants/?name={name}`|https://authorities.smartcosmos.net/tenants/read
+|====================
+
+<<users, *Available methods for users*>>
+
+|====================
+|Endpoint|REST API|Required Authority
+|<<usercreate1, Create>>|`POST /users`|https://authorities.smartcosmos.net/users/create
+|<<userupdate1, Update>>|`PUT /users/{urn}`|https://authorities.smartcosmos.net/users/update
```

Comment:
Not a doc issue :-)
Weird, though. Are you either Admin user or the user whose info is being changed?